app/models.py

```diff
@@ -4,7 +4,9 @@
 
 class User(BaseModel):
     id: str
-    is_user_valid: bool
+    type: str
```

Comment:
Maybe rename this to `AuthUser` so that it's clear. Right now `org` as type of `User` doesn't make sense